mainapp/templates/account.html

```diff
@@ -15,7 +21,46 @@ <h1>Account</h1>
                 <li class="list-group-item">User type: {{ user.user_type_string }}</li>
             </ul>
         </div>
+        <div class="col">
+        {% if user.review_set %}
+            <h1> Submitted Review: </h1>
+
+            <div class="mb-3 text-center location-results">
```

Comment:
Review pane width should be larger

Reply:
![image](https://user-images.githubusercontent.com/18141004/68552499-b22e6300-03e5-11ea-86ab-ee2d7c09d93f.png)
Does it look better?

Reply:
Yes it looks better.

Reply:
@mok232 updated.

Reply:
LGTM!